jdaviz/utils.py

```diff
@@ -81,3 +81,17 @@ def enable_hot_reloading():
         print((
             'Watchdog module, needed for hot reloading, not found.'
             ' Please install with `pip install watchdog`'))
+
+
+def bqplot_clear_context():
+    """Clears hidden context of `bqplot.pyplot` module
+    to mimic matplotlib ``clf()``. This is necessary when
+    we draw multiple plots across different plugins.
```

Comment:
interesting.  Would using the object model instead of pyplot syntax get around this?

Reply:
Maybe? I am not familiar with bqplot at all, so maybe @maartenbreddels can answer. Regardless, I don't have time to do plot refactoring before the end of sprint.

Reply:
I am not excited and code-diving bqplot again but you have a good point and you caught me being lazy. I will see what I can do here.

Reply:
Okay, implemented.